config/StrategyCvxCrvHelperResolver.py

```diff
@@ -47,6 +47,19 @@ def confirm_harvest(self, before, after, tx):
             "sett.pricePerFullShare"
         )
 
+        ## Custom check, bcvxCRV sent funds to badgerTree
+        assert after.balances("bveCVX", "badgerTree") > before.balances("bveCVX", "badgerTree")
```

Comment:
we should probably remove this as we have check on line #54

Reply:
You need both checks to prove that:
1) balance has changed
2) event was triggered with correct values (same as balance change)